cli/docker/app/factory.go

```diff
@@ -30,5 +30,5 @@ func (p *ProjectFactory) Create(c *cli.Context) (project.APIProject, error) {
 
 	context.ProjectName = c.GlobalString("project-name")
 
-	return docker.NewProject(context)
+	return docker.NewProject(context, nil)
```

Comment:
This is more for future (because I think it's safe to merge it that way) but I feel we are not *hugely* consistent with the way we init the `Project` : we pass a lot of *options* via `context`, and some without… The distinction is not that easy/clear… 

I think our end-goal should to have required argumemnts and a struct to pass any options that is not required (or that have some defaults). But it's not really in the PR scope.